src/components/atoms/button/button.scss

```
@@ -0,0 +1,28 @@
+@use "../../variables" as *;
+
+.atoms-button {
+  color: $white;
+  background-color: $black;
+  transition: background-color 0.2s ease;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0px 16px;
+  position: absolute;
+  height: 55px;
+  width: 200px;
+
+  &:hover {
+    background-color: $orange_bistre;
+    color: $white;
+    text-decoration: none;
+  }
+
+  @media screen and(max-width: $breakpoint-md) {
+    height: 45px;
+    width: 180px;
+  }
+  @media screen and(max-width: $breakpoint-lg) {
+    width: 150px;
+  }
```

```suggestion
  @media screen and(max-width: $breakpoint-lg) {
    width: 150px;
  }
  
  @media screen and(max-width: $breakpoint-md) {
    height: 45px;
    width: 180px;
  }
```
順番は逆の方が望ましいです。
今回はデスクトップファーストで書いているので、画面幅が大きい時のスタイル→小さい時のスタイルの順番で書いていくと読みやすいコードになると思います。